### PR TITLE
fix: test PR

### PR DESCRIPTION
**Pull request by tomaszandel**
testowy

### CODE REVIEW

File: ui/apps/ui/src/app/components/filters/filters.service.ts [hunk 1]
```diff
@@ -8,26 +8,77 @@ import {
...
   ) {}
 
-  fetchTreeNodes$(
+  convertFacets(facets: OriginalStructure): ConvertedStructure {
+    const result: ConvertedStructure = {};
+
+    for (const key in facets) {
+      const facet = facets[key];
+      if (typeof facet !== 'number' && 'buckets' in facet) {
+        // Zakładamy, że facet.buckets.map zwraca (string | number)[]
```
Comment (Michal-Kolomanski) [inline ui/apps/ui/src/app/components/filters/filters.service.ts:45]: eng

File: ui/apps/ui/src/app/components/filters/filters.service.ts [hunk 2]
```diff
@@ -8,26 +8,77 @@ import {
...
+        // eslint-disable-next-line @typescript-eslint/no-explicit-any
+        result[key] = facet.buckets.map((bucket: { val: any }) => bucket.val);
+      }
+    }
+
+    return result;
+  }
+
+  /*fetchTreeNodes$(
```
Comment (Michal-Kolomanski) [inline ui/apps/ui/src/app/components/filters/filters.service.ts:54]: Seems like a dev stuff

**Review by Michal-Kolomanski — APPROVED** [review]
From a code perspective it seems like hacking a little bit. The flow got more complicated. However, the most important here are tests. Do we have some examples that on beta don't work, but here they do? @Marcelinna Please test carefully and check whether there are no regressions

File: ui/apps/ui/src/app/components/filters/filters.service.ts (hunk 2 above)
Comment (tomaszandel) [reply ui/apps/ui/src/app/components/filters/filters.service.ts:54]: yup, but i preffer to keep it there at least for a sec

File: ui/apps/ui/src/app/components/filters/filters.service.ts (hunk 1 above)
Comment (tomaszandel) [reply ui/apps/ui/src/app/components/filters/filters.service.ts:45]: yup, will refactor after tests

**Pull request merged**